react/components/Table/Toolbar.js

```diff
@@ -337,6 +340,7 @@ class Toolbar extends PureComponent {
             <ButtonWithIcon
               icon={<IconPlus solid size={LIGHT_ICON_SIZE} />}
               disabled={loading}
+              isLoading={newLine.isLoading}
```

Comment:
Maybe you should add this to the propTypes also.

Reply:
Good, catch, updated.